.github/workflows/version_handling.yml

```diff
@@ -0,0 +1,29 @@
+---
+name: Version-handling check
+
+on:
+  pull_request:
+
+jobs:
+  versions:
+    name: Versions
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        version: ['v0.5.0', 'v0.6.0', 'v0.7.0', 'v0.8.0', 'rc', 'latest']
+    steps:
+      - name: Check out the repository
+        uses: actions/checkout@v2
+
+      - name: Download and install subctl
+        run:
+          VERSION="${{ matrix.version }}" ./getsubctl.sh
+
+      - name: Check we got subctl
+        run:
+          ~/.local/bin/subctl version
+
+      - name: Check we got the right version
+        run: |
+          [ "${{ matrix.version }}" = "latest" ] || ~/.local/bin/subctl version | grep "${{ matrix.version }}"
```

Comment:
You'll probably want to use a more precise comparison for explicit versions, otherwise `v0.8.0-rc1` and such will match `v0.8.0`